### PR TITLE
Change back-to-top icon design

### PR DESCRIPTION
**Pull request by SansPseudoFix**
Screenshot : 
back-to-top4

Global screenshot : 
back-to-top_hover_big

### CODE REVIEW

**Comment by Flaburgan** [comment]
Could you make it square?

**Comment by SansPseudoFix** [comment]
````border-radius: 4px;```` is almost square :P 
Btw, inputs and other stuffs have ````border-radius: 4px;````, this is why I done like that.

**Comment by Flaburgan** [comment]
No, I mean width == height

**Comment by SansPseudoFix** [comment]
Oh, ok. Because, this is the result with ````width: 50px;````: 

back-to-top_width

No matter if I play with paddind or whatever.

**Comment by Flaburgan** [comment]
It actually looks like width > height to me, I would simply add a padding top and bottom to the arrow.

**Comment by SansPseudoFix** [comment]
Like that ?
back-top

You were right btw, I didn't notice bad proportions.

**Comment by Flaburgan** [comment]
Yep. The arrow feels not perfectly vertical-aligned here though.

**Comment by AugierLe42e** [comment]
I don't like the extremities of the arrow. I'd like to see them vertical istead of bended.

**Comment by goobertron** [comment]
Why does this need to be redesigned? I may have missed some discussion somewhere, but I'm quite happy with the current design.

**Comment by AugierLe42e** [comment]
Individual initiative :p

**Comment by SansPseudoFix** [comment]
@goobertron yeah, this is my initiative, because I don't like the current one :P 
But, if nobody wants mine, no problem.

**Comment by goobertron** [comment]
I'm afraid I prefer the existing arrow design. But thanks for trying to improve it!

**Comment by AugierLe42e** [comment]
> I'm afraid I prefer the existing arrow design. 

Don't be afraid, He is not going to eat you (I think...)

**Comment by Flaburgan** [comment]
@SansPseudoFix I'd still like to see the arrow improved. Does your PR removes the underline on hover? I hate that thing :p

File: app/assets/stylesheets/new_styles/_base.scss [hunk 1]
```diff
@@ -147,14 +149,27 @@ $bring-dark-accent-forward-color: #DDD;
   bottom: 20px;
   opacity: 0;
   font-size: 2.9em;
-  padding: 0 12px 0 12px;
-  border-radius: 10px;
+  /*padding: 0 12px 0 12px;*/
```
Comment (Flaburgan) [inline app/assets/stylesheets/new_styles/_base.scss]: Please remove this comment.

File: app/assets/stylesheets/new_styles/_base.scss [hunk 2]
```diff
@@ -147,14 +149,27 @@ $bring-dark-accent-forward-color: #DDD;
...
+  border-radius: 4px;
   background-color: #aaa;
```
Comment (Flaburgan) [inline app/assets/stylesheets/new_styles/_base.scss]: Please use a variable color here. See colors.css

File: app/assets/stylesheets/new_styles/_base.scss [hunk 3]
```diff
@@ -147,14 +149,27 @@ $bring-dark-accent-forward-color: #DDD;
...
+  border-style: solid;
+  border-width: 0.25em 0.25em 0 0;
+  content: '';
+  display: inline-block;
+  width: 20px;
+  height: 20px;
+  position: relative;
+  transform: rotate(-45deg);
+  margin: 0 10px 0 10px;
+  color: #fff;
```
Comment (Flaburgan) [inline app/assets/stylesheets/new_styles/_base.scss]: Please use a variable color here.

**Comment by Flaburgan** [comment]
I'm wondering what's better here, to use a full CSS solution or to use an entypo icon.

**Comment by SansPseudoFix** [comment]
@Flaburgan: No, no more underline on hover. I don't like it too.

IMO, entypo arrow is ugly (or "old design"), that's why I prefer full CSS.

I like blue one too:

back-to-top-blue

File: app/assets/stylesheets/new_styles/_base.scss [hunk 4]
```diff
@@ -69,21 +69,35 @@ a { color : $link-blue  }
...
   z-index: 49;
   right: 20px;
   bottom: 20px;
   opacity: 0;
   font-size: 2.9em;
-  padding: 0 12px 0 12px;
-  border-radius: 10px;
-  background-color: #aaa;
+  border-radius: 4px;
+  background-color: $border-dark-grey;
```
Comment (SansPseudoFix) [inline app/assets/stylesheets/new_styles/_base.scss]: Is it okay to use **border**-dark-grey for **background** ?

**Comment by AugierLe42e** [comment]
I reopen this subject because I've merged this new back-to-top design on diaspote some time ago and I think it works well with the new design:

screen shot 2015-11-29 at 19 25 33

**Comment by denschub** [comment]
Superseded by #6713.

**Pull request closed without merging**